content/advanced_usage/pipelines_as_code.md

```diff
@@ -33,92 +33,42 @@ make it more declarative, depending on the plugin and possibly give more control
 
 ## Currently available plugins for storing pipelines as code
 
-Pipelines can currently be stored using JSON or YAML.
+Pipelines can currently be stored using JSON or YAML. Read more [here](https://github.com/tomzo/gocd-json-config-plugin) and [here](https://github.com/tomzo/gocd-yaml-config-plugin) about what goes in these JSON or YAML files respectively.
```

Comment:
Small changes:

1. Can you link to: https://github.com/tomzo/gocd-json-config-plugin#configuration-files ?

2. Instead of linking the "here", can you link the "JSON" and "YAML" text fragments? Makes it easier.